spec/models/food_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  describe 'Testng food model'
+  subject { Food.new(name: 'Carrot Cake', measurement_unit: 'kg', price: 100) }
+  before { subject.save }
+
+  it 'name should be between 3 - 10 characters' do
+    subject.name = 'this should be false because it is more than 10'
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'price should not be nil' do
+    subject.price = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'price should not be less than 0' do
+    subject.price = -4
+    expect(subject).to_not be_valid
+  end
+
+  it 'measurment should be kg' do

```

Comment:
- [ ] As the previous reviewer mentioned, Please add unit and integration tests using `capybara`.